api/backup/import.go

```diff
@@ -0,0 +1,167 @@
+package backup
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/boltdb/bolt"
+	"github.com/sirupsen/logrus"
+)
+
+// TODO: use portainer-cli to import
+
+func ImportJsonToDatabase(jsonFilePath, portainerDbPath string) error {
+	if _, err := os.Stat(jsonFilePath); err != nil {
+		return fmt.Errorf("import file not found: %s: %s", jsonFilePath, err)
+	}
+	// if _, err := os.Stat(portainerDbPath); err == nil {
+	// 	return fmt.Errorf("ERROR: database file already exists: %s", portainerDbPath)
+	// }
+
+	return importJson(jsonFilePath, portainerDbPath)
```

Comment:
This Stat call is not needed as we can check the error from importJson to determine if the file doesn't exist like.

```
err := importJson(jsonFilePath, portainerDbPath)
if errors.Is(err, os.ErrNotExist) {
    return fmt.Errorf("import file not found: %s: %s", jsonFilePath, err)
} 

return err
```